### PR TITLE
Fix issues 1 - 3

### PR DESCRIPTION
**Pull request by jd147**
here I do make a normal cmd_create command without any function because there is nothing to add.

### CODE REVIEW

**Comment by jd147** [comment]
fix cmd_create function in latest patch and working on test case for jekyll proper installation

File: README.md [hunk 1]
```diff
@@ -1,44 +1,51 @@
...
-Copy repo data into /opt/docker-scripts/: ```cp repodata /opt/docker-scripts/```
+  - Change dir to /opt/docker-scripts/jekyll
 
-Change dir to /opt/docker-scripts/jekyll
+  - Create a directory for the container: `ds init jekyll @jekyll`
 
-Create a directory for the container: ```ds init jekyll @jekyll```
+  - Fix the settings:
+	  ` cd /var/ds/hello/
+	    nano settings.sh`
```
Comment (dashohoxha) [inline README.md]: This is not Ok. What is `/var/ds/hello`?

File: README.md [hunk 2]
```diff
@@ -1,44 +1,51 @@
...
+  - Add to wsproxy: `ds wsproxy add jekyll.example.org`
 
-Add to wsproxy: ```ds wsproxy add jekyll.example.org```
+## Access
 
-To access site use port 4000 with hostname like this ```jekyll.example.org:4000```
+To access site use port 4000 with hostname like this `jekyll.example.org:4000`
 
-Add New project: ```ds @jekyll project-add project```
+## Other Commands
```
Comment (dashohoxha) [inline README.md]: The whole stuff about creating and managing projects, posts etc. can be done in a better, easier and more elegant way.
For example you can share the directory of the projects with the host. Then you can manage the content of the projects from the host.

File: cmd/create.sh [hunk 3]
```diff
@@ -5,5 +5,9 @@ cmd_create_help() {
 
 _EOF
 }
+rename_function cmd_create ds_cmd_create
 
-
+cmd_create(){
+	#there are no log file for jekyll in offical version so no log file mount	
+	ds_cmd_create
+}
```
Comment (dashohoxha) [inline cmd/create.sh]: You can mount the directory of the projects.
If there are any configuration files you can mount them too, or their directory.

**Pull request merged**